pyqtgraph/tests/image_testing.py

```diff
@@ -212,7 +212,7 @@ def assertImageApproved(image, standardFile, message=None, **kwargs):
 
 
 def assertImageMatch(im1, im2, minCorr=None, pxThreshold=50.,
-                       pxCount=0, maxPxDiff=None, avgPxDiff=None,
+                       pxCount=-1, maxPxDiff=None, avgPxDiff=None,
```

Comment:
IIUC `-1` is being used here as a sentinel value.  Why not just let it be `None` like the other defaults?